ReactWindows/ReactNative.Shared/ReactInstanceManager.cs

```diff
@@ -523,10 +523,11 @@ private void SetupReactContext(ReactContext reactContext)
             rootView.TouchHandler = new TouchHandler(rootView);
 
             var jsAppModuleName = rootView.JavaScriptModuleName;
+            var initialProps = rootView.InitialProps;
```

Comment:
If the variable is only used once and contains only one de reference, just inline it.